readme.md

```diff
@@ -198,7 +198,13 @@ Refer to the [caching strategies](#caching-strategy) section for more informatio
 
 ### mem.decorator(options)
 
-Returns a TypeScript decorator which memoizes the given function.
+Returns a [decorator](https://github.com/tc39/proposal-decorators) to memoize class methods.
+
+Notes:
+
+- Only class methods and getters/setters can be memoized, not regular functions (they aren't part of the proposal);
+- Only [TypeScript’s decorators](https://www.typescriptlang.org/docs/handbook/decorators.html#parameter-decorators) are supported, not Babel’s;
```

Comment:
Maybe you can link to Babel decorators as well?

Reply:
```suggestion
- Only [TypeScript’s decorators](https://www.typescriptlang.org/docs/handbook/decorators.html#parameter-decorators) are supported, not [Babel’s](https://babeljs.io/docs/en/babel-plugin-proposal-decorators), which use a different version of the [Decorators Proposal](https://github.com/tc39/proposal-decorators/issues/69);
```